FAILURE: Build failed with an exception.  Required by:hadoop-nna-util:unspecified

need to add maven repo in build.grade
maven { url = "https://plugins.gradle.org/m2/" }

like as:
buildscript {
    repositories {
        maven { url = "https://jcenter.bintray.com" }
        maven { url = "https://repo.maven.apache.org/maven2" }
        maven { url = "https://plugins.gradle.org/m2/" }
    }

-----------------------------------------------------------------------------------------------------------------------------------------------
Build ERROR :
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'hadoop-nna-util'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not find net.nemerosa:versioning:2.7.1.
     Searched in the following locations:
         https://jcenter.bintray.com/net/nemerosa/versioning/2.7.1/versioning-2.7.1.pom
         https://jcenter.bintray.com/net/nemerosa/versioning/2.7.1/versioning-2.7.1.jar
         https://repo.maven.apache.org/maven2/net/nemerosa/versioning/2.7.1/versioning-2.7.1.pom
         https://repo.maven.apache.org/maven2/net/nemerosa/versioning/2.7.1/versioning-2.7.1.jar
     Required by:
         :hadoop-nna-util:unspecified

--- a/build.gradle
+++ b/build.gradle
@@ -26,12 +26,12 @@ apply plugin: 'checkstyle'
 
 buildscript {
     repositories {
-        maven { url = "https://jcenter.bintray.com" }
+        maven { url "https://plugins.gradle.org/m2/" }
         maven { url = "https://repo.maven.apache.org/maven2" }
     }
     dependencies {
-        classpath 'com.netflix.nebula:gradle-ospackage-plugin:2.2.4'
-        classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.10.0'
+        classpath "com.netflix.nebula:gradle-ospackage-plugin:2.2.4"
+        classpath "nl.javadude.gradle.plugins:license-gradle-plugin:0.10.0"
         classpath "com.diffplug.spotless:spotless-plugin-gradle:3.13.0"
         classpath "net.nemerosa:versioning:2.7.1"
     }

